includes/base/element-base.php

```diff
@@ -803,6 +823,14 @@ protected function get_initial_config() {
 		return $config;
 	}
 
+	/**
+	 * A Base method for sanitizing the settings before save.
+	 * This method is meant to be overridden by the element.
+	 */
+	protected function on_save( array $settings ) {
+		return $settings;
```

Comment:
don't you want to add some generic sanitizations?

Reply:
It's an infra only PR